joomla/joomla-cms
# [4.0] Discover string fix

Pull Request for Issue #26299 .

### Summary of Changes

Added strings JLIB_INSTALLER_DISCOVER_INSTALL="Discover Install"
In admin and site lib_joomla.ini

I noticed that there places where strtoupper is missing: e.g  - Text::_('JLIB_INSTALLER_' . $this->route), - leaving that as a separate issue but could fix if anyone would like to confirm - look in libraries/src/Installer/InstallerAdapter.php

### Testing Instructions

See #26299 
I do not know how to test this PR! I checked the code and I am fairly sure there is only one string missing from the set discover_install, install, update and uninstall. 

### Actual result BEFORE applying this Pull Request

Check the code: the strings are missing.

### Expected result AFTER applying this Pull Request

Check the code: the strings are present.

### Documentation Changes Required

None
